misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/NewFormattingTreeModifier.java

```diff
@@ -3630,33 +3635,39 @@ public Token transform(Token token) {
      */
     @SuppressWarnings("unchecked")
     private <T extends Node> T formatNode(T node, int trailingWS, int trailingNL) {
-        if (node == null || node.isMissing()) {
-            return node;
-        }
+        try {
+            if (node == null || node.isMissing()) {
+                return node;
+            }
 
-        if (!isInLineRange(node, lineRange)) {
-            checkForNewline(node);
-            return node;
-        }
+            if (!isInLineRange(node, lineRange)) {
+                checkForNewline(node);
+                return node;
+            }
 
-        int prevTrailingNL = env.trailingNL;
-        int prevTrailingWS = env.trailingWS;
-        env.trailingNL = trailingNL;
-        env.trailingWS = trailingWS;
+            int prevTrailingNL = env.trailingNL;
+            int prevTrailingWS = env.trailingWS;
+            env.trailingNL = trailingNL;
+            env.trailingWS = trailingWS;
 
-        // Cache the current node and parent before format.
-        // Because reference to the nodes will change after modifying.
-        T oldNode = node;
-        Node parent = node.parent();
+            // Cache the current node and parent before format.
+            // Because reference to the nodes will change after modifying.
+            T oldNode = node;
+            Node parent = node.parent();
 
-        node = (T) node.apply(this);
-        if (shouldWrapLine(oldNode, parent)) {
-            node = wrapLine(oldNode, parent);
-        }
+            node = (T) node.apply(this);
+            if (shouldWrapLine(oldNode, parent)) {
+                node = wrapLine(oldNode, parent);
+            }
 
-        env.trailingNL = prevTrailingNL;
-        env.trailingWS = prevTrailingWS;
-        return node;
+            env.trailingNL = prevTrailingNL;
+            env.trailingWS = prevTrailingWS;
+            return node;
```

Comment:
Reduce the two return statements (Within try and the catch blocks) to a single one.

Reply:
Fixed